packages/web-runtime/src/container/api.ts

```diff
@@ -61,22 +61,25 @@ const announceRoutes = (applicationId: string, router: Router, routes: RouteReco
  * inject application specific navigation items into runtime
  *
  * @param applicationId
- * @param store
+ * @param extensionRegistry
  * @param navigationItems
  */
 const announceNavigationItems = (
   applicationId: string,
-  store: Store<unknown>,
+  extensionRegistry: ExtensionRegistry,
   navigationItems: AppNavigationItem[]
 ): void => {
   if (!isObject(navigationItems)) {
     throw new ApiError("navigationItems can't be blank")
   }
 
-  store.commit('SET_NAV_ITEMS_FROM_CONFIG', {
-    extension: applicationId,
-    navItems: navigationItems
-  })
+  const navExtensions = navigationItems.map((navItem) => ({
+    type: 'sidebarNav',
+    navItem,
+    scopes: [applicationId, `app.${applicationId}`]
```

Comment:
Hmm why are both scopes needed?

Reply:
Not necessarily, but it is requested with those 2 types. If we decide for one, I'd go with `app.${applicationId}`. What do you think?

Reply:
Yeah, let's use only one. with prefix sounds good